docs/docs-content/integrations/microk8s.md

```diff
@@ -10,91 +10,275 @@ logoUrl: "https://registry.spectrocloud.com/v1/kubernetes-microk8s/blobs/sha256:
+| -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `microk8sConfig.addons`          | A list of [MicroK8s addons](https://microk8s.io/docs/addons) you can set for your cluster. `Ingress` and `DNS` are mandatory and enabled by default.                                                                           |
+| `microk8sConfig.upgradeStrategy` | It describes how to replace existing machines of your cluster with new ones during upgrades. Values can be `RollingUpgrade` (default), `InPlaceUpgrade`, or `SmartUpgrade`. Refer to the [Usage](#usage) section for guidance. |
+| `microk8sConfig.bootCommands`    | A list of commands you can set to be executed during boot.                                                                                                                                                                     |
+| `microk8sConfig.preRunCommands`  | A list of commands you can set to be executed before installing MicroK8s in your cluster.                                                                                                                                      |
+| `microk8sConfig.postRunCommands` | A list of commands you can set to be executed after installing MicroK8s in your cluster.                                                                                                                                       |
 
-<Tabs queryString="versions">
+### Usage
 
-<TabItem label="1.25.x" value="1.25.x">
+To use MicroK8s, select it as the Kubernetes distribution when choosing the Kubernetes layer during the cluster profile
+creation. Remember that the cloud type must be either AWS or MAAS, and the OS layer must be Ubuntu 22.04.
 
-- **1.25.0**
+![A view of the cluster profile Kubernetes selection screen](/integrations_microk8s_cluster-profile-view.webp)
 
-</TabItem>
+:::info
 
-<TabItem label="1.24.x" value="1.24.x">
+MicroK8s clusters use the [Calico CNI](https://microk8s.io/docs/change-cidr) by default. However, this default
+installation was disabled due to Palette's requirement for a CNI layer to be present in the cluster profiles.
 
-- **1.24.0**
+:::
 
-</TabItem>
-</Tabs>
+#### Upgrade Strategy
 
-MicroK8s installs a minimal, lightweight Kubernetes you can run and use on almost any machine. When installing MicroK8s
-you can specify a channel made up of two components:
+The upgrade strategy describes how to replace existing machines with new ones during upgrades. Three types of upgrade
+strategy are available:
 
-- **Track**: denotes the upstream Kubernetes version.
-- **Risk level**: indicates the maturity level of the release, such as stable and edge.
+- `RollingUpgrade` - This is the default upgrade strategy. It deletes the current control plane machine before creating
+  a new one.
```

Comment:
```suggestion
- `RollingUpgrade` - This is the default upgrade strategy. It deletes the current control plane node before creating
  a new one.
```

Reply:
I'm not sure that _control plane machine_ is the term here, maybe node?

Reply:
:white_check_mark: